submissions/charlesBanda/html-forms/Contact.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
```

Comment:
I noticed in your [7] task, you've added this file, you don't have to added it here, you can remove this file from this branch.

Reply:
Yes, i worked on the signup page and login page. should i add them?

Reply:
NO. Don't add the signup page and login page. They already exist. Your [7] pull request has been merged. You need to delete this file to resolve the conflict on this branch.
@KutrCB